### ISSUE TITLE
Logstash  does'nt send the output to Elasticsearch / synesis lite 1.0.1

### ISSUE THREAD

**Issue by ainayves** [robcowart/synesis_lite_suricata#31]
![image](https://user-images.githubusercontent.com/66997516/125643536-057cb7b2-7d99-4370-958e-92859f8b84b6.png)

Here is tcpdump output from the interface of Elastic , I do not see the ip of logsatsh

**Comment by robcowart** [comment]
You might want to take a look at this video... https://www.youtube.com/watch?v=YA2tGrBQ4v0

If you still have problems you will need to provide much more information. What version of the Elastic Stack components are you using? What is in the logs of both Logstash and Elasticsearch? And please don't provide screenshots of text, as it is very hard to follow and not searchable. You should paste the text into the issue and use format it as `code`.

**Comment by Aina15-DT** [comment]
Hello, I wrote another issue [https://github.com/robcowart/synesis_lite_suricata/issues/32](url)

**Comment by ainayves** [comment]
> You might want to take a look at this video... https://www.youtube.com/watch?v=YA2tGrBQ4v0
> 
> If you still have problems you will need to provide much more information. What version of the Elastic Stack components are you using? What is in the logs of both Logstash and Elasticsearch? And please don't provide screenshots of text, as it is very hard to follow and not searchable. You should paste the text into the issue and use format it as `code`.

Hello,, Thank you for your response Sir , I already watched this video at the beginning,
I wrote another issue in #32
